app/views/referrals/show.html.slim

```diff
@@ -0,0 +1,17 @@
+.govuk-grid-row
+  .govuk-grid-column-two-thirds
+    .govuk-panel.govuk-panel--confirmation.panel-grey
```

Comment:
So in the designs, this box is grey, but for us it's green (I think Lel [wrote some custom CSS](https://github.com/ministryofjustice/eligibility-estimate/blob/main/app/assets/sass/main.scss#L71) to define "panel-grey").

I'm not sure if there is a pre-build grey-background panel component. If not, are we allowed/supposed to write some custom CSS ourselves? Or are we strictly sticking to components that are available to us?

Reply:
I had assumed this was a standard class but I cant find any reference to it. The page this is based on (https://apply-provisional-driving-licence.service.gov.uk/eligibility/dropout_provisional_licence_before?locale=en) uses the 'panel-grey' class but perhaps it is also uses custom CSS. I couldnt find anything in the GDS docs about when or why you should change the panel colour though.

Reply:
added custom css to match the prototype